h2o-py/tests/testdir_algos/automl/pyunit_automl_model_selection.py

```diff
@@ -0,0 +1,259 @@
+from __future__ import print_function
+import sys, os
+
+sys.path.insert(1, os.path.join("..","..",".."))
+import h2o
+import h2o.exceptions
+from tests import pyunit_utils as pu
+from h2o.automl import H2OAutoML
+
+"""This test suite checks the """
+
+max_models = 2
+
+
+def import_dataset(seed=0, larger=False):
+    df = h2o.import_file(path=pu.locate("smalldata/prostate/{}".format("prostate_complete.csv.zip" if larger else "prostate.csv")))
+    target = "CAPSULE"
+    df[target] = df[target].asfactor()
+    #Split frames
+    fr = df.split_frame(ratios=[.8,.1], seed=seed)
+    #Set up train, validation, and test sets
+    return pu.ns(train=fr[0], valid=fr[1], test=fr[2], target=target, target_idx=1)
+
+
+def get_partitioned_model_names(leaderboard):
```

Comment:
@seb-h2o It's not super relevant to current scope but just an observation that we have this method in many py scripts. Maybe we can move some most generic version to utils or somewhere?

Reply:
yes I know, I just copy/paste it, sometimes there are small differences specific for the test, sometimes they're the exact same.
I'm less concerned about abstraction in tests to be honest.

Reply:
@seb-h2o ok, just thought it might be more convenient